src/components/loader-spinner/loader-spinner.pw.tsx

```diff
@@ -168,8 +168,6 @@ test.describe("Prop checks for Loader Spinner component", () => {
     mount,
     page,
   }) => {
-    await page.emulateMedia({ reducedMotion: "no-preference" });
```

Comment:
praise: I like that you have removed this as we should rely on the default browser setting which is to allow motion etc